mycomm/models/vote.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from mycomm.models.article import Article
+
+
+class Vote(models.Model):
+    article = models.OneToOneField(
+        verbose_name='게시글',
+        to=Article,
+        on_delete=models.CASCADE,
+        null=False
+    )
+
+    question = models.CharField(
+        verbose_name='투표 제목',
+        max_length=255,
+        null=False
+    )
+
+    expiredAt = models.DateTimeField(
+        verbose_name='투표 기간 만료 일시',
+        auto_now=False,
+        auto_now_add=False,
+        null=False
+    )
+
+    isExpired = models.BooleanField(
+        verbose_name='투표 기간 만료 여부',
+        null=False
+    )
+
+    answerItems = models.ListCharField(
```

Comment:
```suggestion
    answer_items = models.ListCharField(
```

camelcase -> snake case
모든 필드에 적용해주세요